skins/oasis/modules/PageHeaderController.class.php

```diff
@@ -460,150 +433,106 @@ public function executeEditPage() {
 		}
 
 		// hide floating toolbar when on edit page / in preview mode / show changes
-		if ( $isEdit || $isPreview ) {
-			$wgSuppressToolbar = true;
+		if (
+			$isEdit ||
+			$isPreview ||
+			$isDiff ||
+			( $isHistory && !$this->isUserTalkArchiveModeEnabled )
+		) {
+			// show only "My Tools" dropdown on toolbar
+			$wg->SuppressToolbar = true;
 		}
 
 		// choose header message
 		if ( $isPreview ) {
 			$titleMsg = 'oasis-page-header-preview';
-		}
-		else if ( $isShowChanges ) {
+		} else if ( $isShowChanges ) {
```

Comment:
*¡De nada!* :smiley: Some more here: http://stackoverflow.com/questions/3662412/are-elseif-and-else-if-completely-synonymous